src/julia-parser.scm

```diff
@@ -1060,15 +1066,14 @@
              (if (ts:space? s) (disallowed-space ex t))
              (take-token s)
              (loop (list* 'curly ex (parse-arglist s #\} ))))
-            ((#\")
+            ((#\" #\`)
              (if (and (symbol? ex) (not (operator? ex))
                       (not (ts:space? s)))
-                 ;; custom prefixed string literals, x"s" => @x_str "s"
-                 (let* ((str (begin (take-token s)
-                                    (parse-string-literal s #t)))
+                 ;; custom string and command literals; x"s" => @x_str "s"
+                 (let* ((macstr (begin (take-token s)
```

Comment:
Shouldn't this be `(car (begin ....))` since the previous version also took the `car`?